pkgs/unified_analytics/test/events_with_fake_test.dart

```diff
@@ -30,6 +30,7 @@ void main() {
     description: 'description',
     snoozeForMinutes: 10,
     samplingRate: 1.0, // 100% sample rate
+    excludeDashToolList: [],
```

Comment:
Is there a reason why this field is required? It's fine if it is, I just would have allowed some fields to be optional in order to
- make it easier to author the survey file, and
- minimize the amount of data being sent from the server.